db/migrate/20240404113623_add_decisions_to_application_types.rb

```diff
@@ -6,6 +6,8 @@ class ApplicationType < ActiveRecord::Base; end
   def change
     add_column :application_types, :decisions, :string, array: true
 
+    ApplicationType.reset_column_information
+
```

Comment:
For the record, this can be inside the `up_only` block but it should be fine here as well